source/_components/media_player.samsungtv.markdown

```diff
@@ -59,6 +60,7 @@ Currently known supported models:
 - KS7502 (port must be set to 8001, and `pip3 install websocket-client` must be executed, turn on doesn't work, turn off works fine)
 - K5600AK (partially supported, turn on works but state is not updated)
 - UE65KS8005 (port must be set to 8001, On/Off, Forward/Backward, Volume are OK, but no Play button)
+- MU6300 (port must be set to 8001, `pip3 install websocket-client` must be executed and `get_status_via_rest` must be set to True, or the TV will randomly be turned ON)
```

Comment:
This was already a problem before this refactor, but we should never ask users to install libraries with pip manually. Please add this to the platform requirements.

Reply:
I'm not sure why it was made so, probably because only a handful of TV require this library and this would force the installation for everybody using this component.
Should it still be added as requirement?

Reply:
That would be my preference. Neither option is ideal, but asking users to manually install a library on the terminal is worse.

Reply:
I agree with @armills. This should be in by default.